main.c~

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
These are backups left by your editor.  We could add such common file patterns to .gitignore, but I recommend you to put *~ to [your own git excludesfile](https://github.com/netj/dotfiles/blob/master/.git-excludesfile).